_dashboards/admin-ui-index/index-management.md

```diff
@@ -0,0 +1,223 @@
+
+To close an index, select the index you want to close in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Close** from the drop-down menu.
+
+#### Open
+
+The [open]({{site.url}}{{site.baseurl}}/api-reference/index-apis/open-index/) index operation opens a closed index, letting you add or search for data within the index.
+
+To open an index, select the index you want to open in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Open** from the drop-down menu.
+
+#### Reindex
+
+The [reindex]({{site.url}}{{site.baseurl}}/api-reference/document-apis/reindex/) operation lets you copy all or a subset of your data from a source index into a destination index.
+
+To reindex an index, select the index you want to reindex in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Reindex** from the drop-down menu.
+
+<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/reindex-expanded.png" alt="User interface showing reindex prompt">
+
+#### Shrink
+
+The [shrink]({{site.url}}{{site.baseurl}}/api-reference/index-apis/shrink-index/) index operation moves all of your data in an existing index into a new index with fewer primary shards.
+
+To shrink an index, select the index you want to shrink in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Shrink** from the drop-down menu.
+
+<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/shrink.png" alt="User interface showing shrink prompt">
+
+#### Split
+
+The [split]({{site.url}}{{site.baseurl}}/api-reference/index-apis/split/) index operation splits an existing read-only index into a new index, cutting each primary shard into some amount of primary shards in the new index.
+
+To split an index, select the index you want to split in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Split** from the drop-down menu.
+
+<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/split-expanded.png" alt="User interface showing split page">
+
+#### Delete
+
+If you no longer need an index, you can use the [delete]({{site.url}}{{site.baseurl}}/api-reference/index-apis/delete-index/) index operation to delete it.
+
+To delete an index, select the index you want to delete in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Delete** from the drop-down menu.
+
+## Templates
+
+[Index templates]({{site.url}}{{site.baseurl}}/opensearch/index-templates/) let you initialize new indices with predefined mappings and settings. For example, if you continuously index log data, you can define an index template so that all of these indices have the same number of shards and replicas.
```

Comment:
```suggestion
[Index templates]({{site.url}}{{site.baseurl}}/opensearch/index-templates/) let you initialize new indexes with predefined mappings and settings. For example, if you continuously index log data, you can define an index template so that all of these indexes have the same number of shards and replicas.
```